### DISCUSSION TITLE
Bring back globalSearchLink functionality

### DISCUSSION THREAD

**Discussion by swarley** [laravel/nova-issues#4512] [Ideas & Feature Requests]
In Nova 3 I was able to set an arbitrary `$globalSearchLink` value (in our case 'custom') so that we could link to a resource tool rather than the detail page.

It looks like in nova 4 the only setting that matters is edit, otherwise it's ignored.

Is there any chance we can get this functionality back?

**Comment by davidhemphill** [comment]
In our transition to using Inertia instead of Vue Router for front-end routing, the only custom `linksTo` type you can have is `detail` or `edit`. I think we should modify this to somehow link to a named Inertia page component, but I'm not sure how we should implement that just yet.

**Reply by swarley** [reply]
That would be perfect for us 🙌
